doc/docker/README.md

```diff
@@ -0,0 +1,25 @@
+# Trying out Elektra with Docker
+
+This Dockerfile builds a dockerimage based on ubuntu:xenial including all requirements to build Elektra.
+
+If [Docker](https://www.docker.com/) is available on your machine change to the directory containing the file you are currently reading and build the image with
+```bash
```

Comment:
I've replaced it with sh. I always use _bash_ for highlighting out of a habit, but it should be [synoymous](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L3950).